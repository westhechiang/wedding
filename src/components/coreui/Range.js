import styled from 'styled-components';
import { Range } from 'rc-slider';
import 'rc-slider/assets/index.css';
import { theme } from '../../ui';

const StyledRange = styled(Range)`
  .rc-slider-rail {
    background-color: ${theme.color.grayExtraExtraDark};
  }

  .rc-slider-handle {
    height: 20px;
    width: 20px;
    margin-top: -8px;
    margin-left: -10px;
    border: none;
    background-color: ${theme.color.blueExtraLight};
    background: linear-gradient(
      ${theme.color.blueExtraLight} 0,
      ${theme.color.blueExtraExtraLight} 100%
    );
  }

  .rc-slider-track {
    background-color: #15C0F1;
  }
`;

export default StyledRange;
