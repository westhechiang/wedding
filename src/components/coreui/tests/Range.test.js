import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Range from '../../../components/coreui/Range';

describe('<Range />', () => {
  it('should match its empty snapshot', () => {
    const wrapper = shallow(<Range />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
