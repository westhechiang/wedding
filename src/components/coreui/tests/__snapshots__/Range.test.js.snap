// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Range /> should match its empty snapshot 1`] = `
<ComponentEnhancer(Range)
  activeDotStyle={Object {}}
  allowCross={true}
  className="sc-bdVaJa iFsqIN"
  count={1}
  disabled={false}
  dotStyle={Object {}}
  dots={false}
  handle={[Function]}
  handleStyle={
    Array [
      Object {},
    ]
  }
  included={true}
  marks={Object {}}
  max={100}
  min={0}
  onAfterChange={[Function]}
  onBeforeChange={[Function]}
  onChange={[Function]}
  prefixCls="rc-slider"
  pushable={false}
  railStyle={Object {}}
  step={1}
  trackStyle={
    Array [
      Object {},
    ]
  }
  vertical={false}
/>
`;
